components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -17,28 +17,31 @@
 #import "MDCFeatureHighlightViewController.h"
 
 #import <MDFTextAccessibility/MDFTextAccessibility.h>
+#import "MaterialFeatureHighlightStrings.h"
+#import "MaterialFeatureHighlightStrings_table.h"
 #import "MaterialTypography.h"
 #import "private/MDCFeatureHighlightAnimationController.h"
 #import "private/MDCFeatureHighlightView+Private.h"
 
+// The Bundle for string resources.
+static NSString *const kMaterialFeatureHighlightBundle = @"MaterialFeatureHighlight.bundle";
+
 static const CGFloat kMDCFeatureHighlightLineSpacing = 1.0f;
 static const CGFloat kMDCFeatureHighlightPulseAnimationInterval = 1.5f;
 
 @interface MDCFeatureHighlightViewController () <UIViewControllerTransitioningDelegate>
-@property(nonatomic, strong) MDCFeatureHighlightView *view;
+@property(nonatomic, nullable, weak) MDCFeatureHighlightView *featureHighlightView;
 @end
```

Comment:
Why is this weak now?

Reply:
Originally we had a strong reference to MDCCustomView that was aliasing the standard UIView *view.
This can cause confusion for developers as we are invoking methods and properties on our customView that don't exist in UIView.
To put it another way:
How are we setting self.view.titleFont = XYZ when UIView doesn't have a titleFont property?

Reply:
I'm not so concerned about the type of the object as much as why it's `weak`. Why does it have to be `weak`?

Reply:
E.g. UIViewController's `.view` type is:

```objc
@property(null_resettable, nonatomic,strong) UIView *view;
```